src/js/actions/MesosActions.js

```diff
@@ -0,0 +1,48 @@
+import AppDispatcher from "../AppDispatcher";
```

Comment:
Could you still use the old require-function to be consistent with our old code.
That has the reason, not to switch between syntax if you work on 2 files with old and new syntax.

I will create a tech debt issue to refactor the require to the new import-statement all at once.